File: C/Easy/Jack-O'-Lantern Juxtaposition/Jack_O_Lantern.c
#include <stdio.h>

int main()
{
    long long eyes, nose, mouths;
    while (scanf("%lld%lld%lld", &eyes, &nose, &mouths) == 3)
    {
        int result = eyes * nose * mouths;
        printf("%lld\n", result);

    }
}

File: Java/Easy/Jack-O'-Lantern Juxtaposition/Jack_O_Lantern.java
import java.util.Scanner;

/**
 * Jack_O_Lantern
 */
public class Jack_O_Lantern {

    public static void main(String[] args) {
        try {
            long eyes, nose, mouths;
            Scanner sc = new Scanner(System.in);

            while (sc.hasNextLong()) {
                eyes = sc.nextLong();
                nose = sc.nextLong();
                mouths = sc.nextLong();

                long result = eyes * nose * mouths;

                System.out.println(result);
            }
            sc.close();

        } catch (Exception e) {
            System.out.println("Something happend \n");
            e.printStackTrace();
        }
    }
}